### ISSUE TITLE
Update storybook a11y addon config

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/boiler#1089]
### Description / User story

The issue reported in #1088 should have been reported by the storybook a11y addon as well, but was not as our config did not include the needed rule.

This should be fixed with this task.

We could use the following config for this:
`        a11y: {
            // axe-core options
            options: {
                // Even though this says "runOnly", it is used here to enable
                // more rules than the ones enabled by default
                runOnly: {
                    type: "tag",
                    // Run all WCAG 2.x rules & best practices, level A & AA
                    values: [
                        "wcag2a",
                        "wcag2aa",
                        "wcag21a",
                        "wcag21aa",
                        "wcag22aa",
                        "best-practice",
                        // Consider disabling experimental if it repeatedly
                        // causes issues.
                        // For singular issues, deactivate bad rules per-story.
                        "experimental",
                    ],
                },
            },
        },`

### Requirements / Prerequisites

- none

### Acceptance Criteria

- [x] The storybook a11y addon config has been updated and now includes the rule that would have found the issue of #1088

### Additional information

- The config above was shared by @remmycat . In case there are questions they might be able to help us.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
